### PR TITLE
feat(template) : create default template for pr

### PR DESCRIPTION
**Pull request by hussamkhatib**
This PR fixes issue #9

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This PR fixes issue #__  
+Description of the change:
```
Comment (souvikbasu) [inline .github/pull_request_template.md:2]: Also, add a line for "Screenshot (if applicable):"

**Review by souvikbasu — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (souvikbasu) [reply .github/pull_request_template.md:2]: Add Screenshot before Description

**Comment by hussamkhatib** [comment]
Screenshot from 2021-05-22 18-54-02

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+This PR fixes issue #__  
+Screenshot (if applicable)
```
Comment (souvikbasu) [inline .github/pull_request_template.md:2]: need a colon at the end like in Description:

**Review by souvikbasu — CHANGES_REQUESTED** [review]

**Review by souvikbasu — APPROVED** [review]

**Pull request merged**